platformcomponents/desktop/messages.json

```diff
@@ -0,0 +1,52 @@
+{
+  "message": {
+    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=5712%3A57",
+    "comment": "Color tokens for a conversation space",
+    "bodyText-primary": {
+      "#normal": {
+        "text": "@theme-text-primary-normal"
+      }
+    },
+    "bodyText-secondary": {
+      "#normal": {
+        "text": "@theme-text-secondary-normal"
+      }
+    },
+    "bodyText-hyperlink": {
+      "#normal": {
+        "text": "@theme-text-accent-normal"
+      }
+    },
+    "bodyText-mentionSelf": {
+      "#normal": {
+        "text": "@theme-text-accent-normal"
+      }
+    },
+    "bodyText-mentionOther": {
+      "#normal": {
+        "text": "@theme-text-secondary-normal"
+      }
+    },
+    "bodyText-mentionAll": {
+      "#normal": {
+        "text": "@theme-text-accent-normal"
+      }
+    },
+    "extParticipants": {
+      "#normal": {
+        "text": "@theme-text-warning-normal"
+      }
+    },
+    "banner": {
```

Comment:
we should create a different component for this called 'chat-new-message'. That means a different file. There, we should put all the colours we find here: https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=41682%3A109556